### DISCUSSION TITLE
React Native ?

### DISCUSSION THREAD

**Discussion by rick427** [SupremeTechnopriest/react-idle-timer#364] [General]
Hi, can I use this lib with react-native ?

**Comment by SupremeTechnopriest** [comment]
There's no official support, but I think it should work. I believe there are some RN projects using it. The only thing I can think of is cross tab and worker timers wouldn't work, but vanilla it should be good to go!

**Reply by rick427** [reply]
Thanks for replying, I'm happy to hear that it can be used in a RN project, however after installing it earlier i had a error saying "cannot find document" which made me realize it can only be used on web, maybe am doing something wrong ...

**Reply by SupremeTechnopriest** [reply]
Try setting the element prop to a ref.
